flashcart-notes.txt

```diff
@@ -0,0 +1,91 @@
+flashcarts that can be directly built:
```

Comment:
This file would IMO make more sense as a wiki, rather than something distributed directly with the repository.

Reply:
Removed it for now.